src/components/Inputs/Label/Label.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Label as UIFabLabel } from 'office-ui-fabric-react/lib/Label';
+
+/** Label example */
+const label ="I am a label";
+function Label({required, disabled }) {
+	return (<UIFabLabel required={required} disabled={disabled} >{label}</UIFabLabel>);
+}
+
+Label.propTypes = {
```

Comment:
Add the label property as a required propType of type string.

Reply:
Yeah